src/registrar/assets/sass/_theme/_admin.scss

```
@@ -158,8 +158,10 @@ table > caption > a {
     padding-top: 20px;
 }
 
-// 'Delete button' layout bug
-.submit-row a.deletelink {
+// Fix django admin button height bugs
+.submit-row a.deletelink,
+.delete-confirmation form .cancel-link,
```

```suggestion
.delete-confirmation form.cancel-link,
```
Did you mean to add a space here? This will apply to all `<form>`(s) otherwise